javascript/javascript1/week3/activities.js

```diff
@@ -0,0 +1,44 @@
+let activities = [];
+let date = new Date();
+function addActivity(date, activity, duration) {
+  activities.push({ date, activity, duration });
+  return;
+}
+addActivity(date, "Youtube", 40);
+addActivity(date, "facebook", 30);
+addActivity(date, "snapchat", 20);
+console.log(activities);
+
+function showStatus() {
+  let total = 0;
+  for (let x = 0; x < activities.length; x++) {
+    total = total + activities[x].duration;
+  }
+  console.log(
+    "You have added " +
+      activities.length +
+      " activities. They amount to " +
+      total +
+      " min. of usage"
+  );
+  if (total > 60) {
+    console.log("You have reached your limit, no more smartphoning for you!");
+  }
+  if (typeof activities == undefined) {
+    console.log("Add some activities before calling showStatus");
+  }
```

Comment:
it is better to do this check at the beginning of the function, and because `activities` is an array, just check the length
```js
    function showStatus() {
    if (activities.length === 0) {
      console.log('Add some activities before calling showStatus');
      return;
    }
```
`return` at this condition will stop executing the function. But the one you added at the end of function has no meaning.